packages/carousel/src/Carousel.js

```diff
@@ -8,6 +8,7 @@ import { Dots } from './Dots'
 import { ArrowButton } from './ArrowButton'
 import { Slider } from './Slider'
 import { getSlideKey, getVisibleSlidesArray, useResponsiveVisibleSlides } from './helpers'
+import { StretchSlide } from './StretchSlide/StretchSlide'
```

Comment:
n-b mind adding an `index` file so you can omit `/StretchSlide`?

Reply:
Updated